neurons/competition_runner.py

```diff
@@ -8,124 +7,47 @@
 from types import SimpleNamespace
 from datetime import datetime, timezone, timedelta
 import bittensor as bt
-from typing import List
-
-from competition_config import competitions as competitions_cfg
-
-import wandb
+from typing import List, Tuple, Dict
```

Comment:
I think python 3.10 supports those types natively, you can use for example list[something]

Reply:
no harm